src/Propel/Generator/Reverse/PgsqlSchemaParser.php

```diff
@@ -633,6 +668,9 @@ protected function addSequences(Database $database): void
             AND c.relkind = 'S'
             AND n.nspname IN ($searchPath);
         ");
+        if ($stmt === false) {
+            throw new RuntimeException('prepare() returned no statement.');
```

Comment:
Uff, thank you for adding all those! Might make sense to change `prepare()` to throw an exception instead of doing the stupid `false` return